packages/utils/node-resolver-rs/src/lib.rs

```diff
@@ -158,6 +164,16 @@ impl<'a, Fs: FileSystem> Resolver<'a, Fs> {
     specifier: &'s str,
     from: &Path,
     specifier_type: SpecifierType,
+  ) -> ResolveResult {
+    self.resolve_with_options(specifier, from, specifier_type, Default::default())
+  }
+
+  pub fn resolve_with_options<'s>(
+    &self,
+    specifier: &'s str,
+    from: &Path,
+    specifier_type: SpecifierType,
+    options: ResolveOptions,
```

Comment:
Is there a case where there you'd want no conditions to apply (and then this should be an `Option<ResolveOptions>`)? Probably not..

Reply:
You mean to disable the import/require conditions but not add any in its place? Hmm... not sure.